Packs/ExtraHop/README.md

```diff
@@ -1,3 +1,71 @@
+<~XSIAM>
+ # Extrahop RevealX
+ ## Overview
+ExtraHop RevealX 360 is a cloud-based security platform that provides comprehensive visibility and advanced threat detection across your network. It helps organizations quickly identify and respond to security threats in hybrid and multi-cloud environments. By leveraging machine learning and behavioral analytics, RevealX 360 monitors network traffic to spot suspicious activities and potential risks in real time.
+
+## This pack includes:
+- Rest API log collection for detection logs
+- Modeling Rules for detection logs
+
+## Configure an instance for ExtraHop Reveal(x)
+
+### How to create REST API Credentials:
+* You must have system and access administration privileges.
+1. Log in to RevealX 360.
+2. Click the System Settings icon - at the top right of the page and then click All Administration.
+3. Click API Access.
+4. Click Create Credentials.
+5. In the Name field, type a name for the credentials.
+6. In the Privileges field, specify a privilege level for the credentials. The privilege level determines which actions can be performed with the credential. Do not grant more privileges to REST API credentials than needed because it can create a security risk. For example, applications that only retrieve metrics should not be granted credentials that grant administrative privileges. For more information about each privilege level, see User privileges. 
```

Comment:
```suggestion
2. Click the **System Settings** icon at the top right of the page and then click **All Administration**.
3. Click **API Access**.
4. Click **Create Credentials**.
5. In the Name field, type a name for the credentials.
6. In the Privileges field, specify a privilege level for the credentials. The privilege level determines which actions can be performed with the credential. Do not grant more privileges to REST API credentials than needed because it can create a security risk. For example, applications that only retrieve metrics should not be granted credentials that grant administrative privileges. For more information about each privilege level, see User privileges. 
```
Where is User privileges? Can you add a link?